resources/prometheus/prometheus-rules.yaml

```diff
@@ -358,8 +358,8 @@ spec:
 
         - expr: |
             sum by (namespace, rhacs_instance_id, rhacs_org_id, rhacs_org_name, rhacs_cluster_name, rhacs_environment)
-            (rate(grpc_server_started_total{namespace=~"rhacs-.*", job="central", grpc_type="unary",grpc_service!="v1.PingService"}[10m]))
-          record: central:grpc_server_started:total:rate10m
+            (rate(grpc_server_handled_total{namespace=~"rhacs-.*", job="central", grpc_type="unary",grpc_service!="v1.PingService"}[10m]))
```

Comment:
Not related to this PR, but it might be worthwhile to consider ignoring the `v1.MetadataService` here as well.

Reply:
I'm not so sure 😅 . See https://github.com/stackrox/rhacs-observability-resources/pull/134 while ignoring it makes sense, it also creates problems because the traffic to many Centrals is so sparse.